lib/factories.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Bigben.Factories do
```

Comment:
I wondered why you done that. I don't like calling it `Factory`. I'm going to try this and see if I care after the next project. Thanks for the explanation though.